erpnext/setup/setup_wizard/operations/taxes_setup.py

```diff
@@ -292,7 +292,7 @@ def get_or_create_tax_group(company_name, root_type):
 
 	tax_group_account.flags.ignore_links = True
 	tax_group_account.flags.ignore_validate = True
-	tax_group_account.insert(ignore_permissions=True)
+	tax_group_account.insert(ignore_permissions=True, ignore_if_duplicate=True)
```

Comment:
Allowing duplicate accounts in COA doesn't look like a good idea.

Reply:
@Sanket322 

Maybe a validation method to ensure correct company address is maintained, If not give a popup to clear and reset it?

Reply:
> Allowing duplicate accounts in COA doesn't look like a good idea.
I think it won’t create a duplicate account; rather, it will ignore the request to create the document.

This is called during india compliance installation so we even can not use any popup.